test/test-signatures.spec.ts

```diff
@@ -1,22 +1,25 @@
 import { SAML } from "../lib/passport-saml/index.js";
 import * as fs from "fs";
 import * as sinon from "sinon";
+import { SamlOptions } from "../lib/passport-saml/types.js";
 import assert = require("assert");
 
 const cert = fs.readFileSync(__dirname + "/static/cert.pem", "ascii");
 
 describe("Signatures", function () {
   const INVALID_SIGNATURE = "Invalid signature",
+    INVALID_ENCRYPTED_SIGNATURE = "Invalid signature from encrypted assertion",
     createBody = (pathToXml: string) => ({
       SAMLResponse: fs.readFileSync(__dirname + "/static/signatures" + pathToXml, "base64"),
     }),
     testOneResponseBody = async (
       samlResponseBody: Record<string, string>,
       shouldErrorWith: string | false | undefined,
-      amountOfSignatureChecks = 1
+      amountOfSignatureChecks = 1,
+      options: Partial<SamlOptions> = {}
```

Comment:
Why not use the type `SamlConfig` instead as that better represents what a consumer should be using.

Reply:
Good suggestion. I've updated the code.